doc/migrations/v0.46-v1.0.md

```diff
@@ -64,6 +64,22 @@ const libp2p = await createLibp2p({
 const keychain: KeyChain = libp2p.services.keychain
 ```
 
+### AutoNAT
```

Comment:
I think we should move all these changes to the [v0.46-v1.0.0.md](https://github.com/libp2p/js-libp2p/blob/release-v1.0/doc/migrations/v0.46-v1.0.0.md) file